iphone/TitaniumKit/TitaniumKit/Sources/Kroll/KrollObject.m

```diff
@@ -148,9 +148,6 @@ void KrollInitializer(JSContextRef ctx, JSObjectRef object)
 
   if ([o isKindOfClass:[KrollObject class]]) {
     [o retain];
-    JSObjectRef propsObject = JSObjectMake(ctx, NULL, NULL);
-    JSObjectSetProperty(ctx, object, kTiStringTiPropertyKey, propsObject, kJSPropertyAttributeDontEnum, NULL);
-    [o setPropsObject:propsObject];
```

Comment:
@janvennemann Why removed?

Reply:
This will now be created lazily in the getter for `propsObject`. The js object only needs to be created once `jsObject` or `propsObject` are used so i moved everything related to creating those objects into the corresponding getter.